notebook/static/notebook/js/cell.js

```diff
@@ -299,6 +299,7 @@ define([
                     this.element.removeClass('marked');
                 }
             }
+            this.events.trigger('marking_changed.Cell', {cell:this, value:value});
```

Comment:
Usually when we have an attribute named `foo` the change method would be called `foo_changed`. I think I would prefer to have all the events/methods named with `marked` rather than `marking`.